### PR TITLE
Remove feature tests run as follow up to #913

### PR DESCRIPTION
**Pull request by edd**
After #913, I think I saw an error in the build due to this PR. Now it shouldn’t.

### CODE REVIEW

**Comment by cpuja84** [comment]
Hi @edd - I already raised this issue with @fkondej who would be looking at it tomorrow ( 17/02/2022). Thanks !

**Comment by edd** [comment]
@cpuja84 🤞 I got it right and have saved him some work. But I did leave a single test running in parallel so I doubt I got it exactly right. You’ve assigned @fkondej to it now so I’ll wait for his review 😄

**Review by fkondej — APPROVED** [review]
More cleanup is required. But this should unblock other PR checks.

**Pull request merged**
